packages/hardhat-cannon/src/builder/storage.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import AdmZip from 'adm-zip';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { ChainDefinition } from './types';
+
+export function getCacheDir(cacheFolder: string, name: string, version: string) {
+  return path.join(cacheFolder, 'cannon', name, version);
+}
+
+export function getLayerFiles(cacheDir: string, chainId: number, n: number) {
+  const filename = `${chainId}-${n}`;
+
+  const basename = path.join(cacheDir, filename);
+
+  return {
+    cannonfile: path.join(cacheDir, 'cannonfile.json'),
+    chain: basename + '.chain',
+    metadata: basename + '.json',
+  };
+}
+
+export async function exportChain(hre: HardhatRuntimeEnvironment, name: string, version: string): Promise<Buffer> {
+  const zip = new AdmZip();
+
+  await zip.addLocalFolderPromise(getCacheDir(hre.config.paths.cache, name, version), {});
+  zip.addLocalFile(path.join(hre.config.paths.root, 'README.md'));
```

Comment:
The `README.md` file could not exists, could break in this case

Reply:
7e5372e